src/Inputs/UploadInput.php

```diff
@@ -47,7 +47,7 @@ public function getControl()
 	{
 		/** @var Html $control */
 		$control = parent::getControl();
-		$control->class = 'custom-file-input';
+		trim($control->class .= ' custom-file-input');
```

Comment:
trim returns string, does not have any side effects. In this case return value is not assigned to anything... This code does not make sense...

Reply:
yeah i know and i am idiot...

Reply:
it should be ok now